particles/MeshParticles.cpp

```diff
@@ -21,29 +18,13 @@ TexturedMeshForParticles::TexturedMeshForParticles(std::string meshName)
     vertexInputBindings.push_back({4 * sizeof(float), QRhiVertexInputBinding::PerInstance});
     vertexAttributeBindings.push_back(
                 {2, 2, QRhiVertexInputAttribute::Float4, 0});
-
-    vertexArray = mesh;
-    vertexCount = myData.vertices_length / 3;
 }
 
-TexturedMeshForParticles& TexturedMeshForParticles::instance(std::string meshName) noexcept
+void TexturedMeshForParticles::setMesh(std::vector<float>& mesh, meshdata& myData)
```

Comment:
Added this function to set the counts when the mesh is loaded although I believe it's actually unnecessary since setupBindings was removed